policyengine_us/variables/gov/ed/pell_grant/head/pell_grant_head_available_income.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class pell_grant_head_available_income(Variable):
+    value_type = float
+    entity = Person
```

Comment:
```suggestion
    entity = Person
    unit: USD
```

Reply:
Could you add the unit to all variables (besides bool and ENUMs) - just for out app display

Reply:
What unit should I use for:

- pell_grant_dependents_in_college
- pell_grant_months_in_school

Reply:
we can skip units for each of those